apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var _ = require('lodash');
+var color = require('../../color');
+
+// TODO - better handle packages like marked (perhaps using browserify-shim).
+// I'd also like to have a centralized place across all of our mochaTests where
+// we initialize all globals we depend on dashboard/code-studio for (jquery,
+// react, marked, etc.).
+var processMarkup = window.marked || function (str) { return str; };
```

Comment:
This is pretty gross, but properly shimming marked (and cleaning up how we handle globals in our tests in general) is a bigger item, that I'd rather handle in a separate PR.